sdk/core/Azure.Core/tests/DynamicData/MutableJsonDocumentTests.cs

```diff
@@ -165,61 +166,6 @@ public void JsonWithDelimiterIsInvalidJson()
             Assert.IsTrue(threwJsonException);
         }
 
-        [Test]
-        public void CanSetPropertyToMutableJsonDocument()
```

Comment:
These are no longer relevant since setting to MutableJsonDocument is not supported.